<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rectifierForm</class>
 <widget class="QWidget" name="rectifierForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Rectifer Settings</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="volt_LineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>51</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="current_LineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>121</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rectifierSettingButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>VOLT</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <zorder>label_15</zorder>
   <zorder>label_18</zorder>
   <zorder>volt_LineEdit</zorder>
   <zorder>current_LineEdit</zorder>
   <zorder>rectifierSettingButton</zorder>
   <zorder>eightButton</zorder>
   <zorder>fiveButton</zorder>
   <zorder>zeroButton</zorder>
   <zorder>twoButton</zorder>
   <zorder>sixButton</zorder>
   <zorder>oneButton</zorder>
   <zorder>selectButton</zorder>
   <zorder>nineButton</zorder>
   <zorder>sevenButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>threeButton</zorder>
   <zorder>fourButton</zorder>
  </widget>
  <widget class="QPushButton" name="rectifierbackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
